packages/mobile/src/redux/migrations.ts

```diff
@@ -557,4 +557,11 @@ export const migrations = {
       },
     }
   },
+  41: (state: any) => ({
+    ...state,
+    app: {
+      ...state.app,
+      dappsWebViewEnabled: false,
+    },
+  }),
```

Comment:
We can also simply return the state. The state reconciler will add the new prop as it merges it with the new initialState.
But good to be explicit too. 👍 

```suggestion
  41: (state: any) => state,
```

Reply:
i made this change :) TBH i don't have strong feelings about it though - i know adding properties can be done automatically like this so it works, but removing properties we need to do manually... so it's an extra thing that we need to remember (rather than just blanket rule for explicitly declaring what you are doing to the redux state in a migration)? consistency is a little hard to enforce as well, although not so important for this case i guess 😅